admins/pageflow/entry.rb

```diff
@@ -50,28 +59,48 @@ module Pageflow
     filter :edited_at
     filter :first_published_at
     filter :published_revision_published_at, as: :date_range
-    filter :with_publication_state, as: :select, collection: -> { collection_for_entry_publication_states }
-
-    sidebar :folders, :only => :index do
-      text_node(link_to(I18n.t('pageflow.admin.entries.add_folder'), new_admin_folder_path, :class => 'new'))
-      grouped_folder_list(Folder.includes(:account).accessible_by(Ability.new(current_user), :read),
-                          :class => authorized?(:manage, Folder) ? 'editable' : nil,
-                          :active_id => params[:folder_id],
-                          :grouped_by_accounts => authorized?(:read, Account))
+    filter :with_publication_state,
+           as: :select,
+           collection: -> { collection_for_entry_publication_states }
+
+    sidebar :folders, only: :index do
+      text_node(link_to(I18n.t('pageflow.admin.entries.add_folder'),
+                        new_admin_folder_path,
+                        class: 'new'))
+      grouped_folder_list(Folder.accessible_by(Ability.new(current_user), :read),
+                          active_id: params[:folder_id],
+                          grouped_by_accounts: true)
     end
 
     form do |f|
       f.inputs do
-        f.input :title, :hint => I18n.t('pageflow.admin.entries.title_hint')
-        if authorized?(:read, Account)
-          f.input :account, :include_blank => false
+        f.input :title, hint: I18n.t('pageflow.admin.entries.title_hint')
 
-          unless f.object.new_record?
-            f.input :theming, :include_blank => false
-          end
+        eligible_accounts = AccountPolicy::Scope.new(current_user, Pageflow::Account)
+                            .entry_movable.load
+        if authorized?(:update_account_on, resource)
+          f.input :account,
+                  collection: eligible_accounts,
+                  include_blank: false,
+                  input_html: {onchange: jquery_for_disabling_on_change}
```

Comment:
Let's use "progressive enhancement"-style JS and move this inline javascript to `pageflow/admin/entries.js`.